JS/utils.js

```diff
@@ -0,0 +1,8 @@
+
+function print(planes) {
```

Comment:
Please give more meaningful name to the function.

Reply:
Done. 'Print' was renamed into 'printObjectToString'